config/settings.yml

```diff
@@ -30,4 +30,4 @@ database:
   poolsize: "_env:PGPOOLSIZE:10"
 
 copyright: "thoughtbot, inc. 2015"
-#analytics: UA-YOURCODE
+analytics: "nMWJWvFASCtQm0dr3nIZLDszgHvm7vhx"
```

Comment:
This is a write-only code. It's also going into the page source (by design), so I don't see putting it in the repo as a problem.

Please correct me if I'm wrong.